test/util/testfactory/utils.go

```diff
@@ -132,22 +132,6 @@ func NewBaseAccount(kr keyring.Keyring, name string) (*authtypes.BaseAccount, sd
 	return bacc, origCoins
 }
 
-// GenerateBaseAccounts generates base accounts and their corresponding balances.
-// It takes a keyring.Keyring instance and a slice of account names as inputs.
-// For each name in the names slice, it creates a new base account and adds it to
-// the bAccounts slice. It also creates a banktypes.Balance struct for the account,
-// with the account's address and coins including an initial coin for gas fee.
-// The balances are added to the balances slice.
-//
-// Parameters:
-// - kr: A keyring.Keyring instance used to create base accounts.
-// - names: A slice of account names.
-// - initExtraCoins: A slice of sdk.Coins representing the extra coins to be added to
-// the base accounts which can be used for example for gas fees.
-//
-// Returns:
-// - bAccounts: A slice of authtypes.BaseAccount representing the generated base accounts.
-// - balances: A slice of banktypes.Balance representing the balances of the generated accounts.
```

Comment:
[not blocking][question] why were all the comments in this file removed?

Reply:
Just to clean up as the function signatures were descriptive enough